api/database/seeders/TalentNominationTestSeeder.php

```diff
@@ -49,5 +52,18 @@ public function run()
             ->submittedReviewAndSubmit()
             ->create();
 
+        // seed some to a specific user
+        TalentNomination::factory()
+            ->count(1)
+            ->noSubmittedSteps()
+            ->create([
+                'submitter_id' => $admin->id,
+            ]);
+        TalentNomination::factory()
+            ->count(1)
+            ->submittedReviewAndSubmit()
+            ->create([
+                'submitter_id' => $admin->id,
+            ]);
```

Comment:
Two to start, but any specific cases people want to test would be good to append here whenver